styles/components/_view.scss

```diff
@@ -30,10 +30,10 @@
   justify-content: space-between;
 }
 .button {
-  display: block;
-  width: 120px;
-  height: 30px;
-  margin: 0 auto;
+  // display: block;
+  // width: 120px;
+  // height: 30px;
+  // margin: 0 auto;
```

Comment:
Why is it in the comment? Is it necessary right now?

Reply:
There are properties that were set for release, but will have to be removed / rethought